### PR TITLE
Restructure frontend components

### PR DESCRIPTION
**Pull request by tombui99**

### CODE REVIEW

File: ui/src/components/miscComponents/SocialLogin.svelte [hunk 1]
```diff
@@ -1,11 +1,7 @@
 <script>
   import firebase from "firebase/app";
   import "firebase/auth";
-  import LoadingCirle from "../components/LoadingCirle.svelte";
-  import { createEventDispatcher, onMount } from "svelte";
-  import { Navigate } from "svelte-router-spa";
-  import TextField from "../components/TextField.svelte";
-  import { oauthLoginError } from "../stores.js";
+  import LoadingCirle from "./LoadingCirle.svelte";
```
Comment (bfabiny) [inline ui/src/components/miscComponents/SocialLogin.svelte:4]: Type - change 
From 
    LoadingCirle
To 
    LoadingCircle

File: ui/src/containers/ArtilleryReport.svelte [hunk 2]
```diff
@@ -1,26 +1,21 @@
...
+  import Breadcrumbs from "../components/miscComponents/Breadcrumbs.svelte";
+  import LoadingFlat from "../components/miscComponents/LoadingFlat.svelte";
+  import Tabs from "../components/miscComponents/Tabs.svelte";
+  import Icon from "../components/miscComponents/Icon.svelte";
+  import Toastr from "../components/miscComponents/Toastr.svelte";
   import slug from "slug";
-  import { format } from "date-fns";
-  import formatDistanceToNow from "date-fns/formatDistanceToNow";
-  import ArtilleryChart from "../components/ArtilleryChart.svelte";
+  import ArtilleryChart from "../components/artileryComponents/ArtilleryChart.svelte";
```
Comment (bfabiny) [inline ui/src/containers/ArtilleryReport.svelte]: Typo in the folder name
Change
From
    artileryComponents
To
    artilleryComponents

File: ui/src/containers/Dashboard.svelte [hunk 3]
```diff
@@ -2,10 +2,10 @@
   import { userApi, userSession$ } from "../stores";
   import { onDestroy } from "svelte";
   import marked from "marked";
-  import Icon from "../components/Icon.svelte";
+  import Icon from "../components/miscComponents/Icon.svelte";
   import firebase from "firebase/app";
-  import BuildList from "../components/BuildList.svelte";
-  import LoadingFlat from "../components/LoadingFlat.svelte";
+  import BuildList from "../components/buildListCardComponent/BuildList.svelte";
```
Comment (bfabiny) [inline ui/src/containers/Dashboard.svelte]: The folder names should be consistent. 
Change
From 
    buildListCardComponent
To
    buildListCardComponents

File: ui/src/containers/HtmlHints.svelte [hunk 4]
```diff
@@ -5,25 +5,18 @@
...
-  import HtmlErrorsTable from "../components/HtmlErrorsTable.svelte";
-  import Breadcrumbs from "../components/Breadcrumbs.svelte";
-  import slug from "slug";
-  import Toastr from "../components/Toastr.svelte";
-  import BuildDetailsCard from "../components/BuildDetailsCard.svelte";
-  import { CONSTS, getPerfScore, HTMLERRORS, printTimeDiff } from "../utils/utils.js";
+  import HtmlErrorsTable from "../components/htmlhintComponents/HtmlErrorsTable.svelte";
+  import Breadcrumbs from "../components/miscComponents/Breadcrumbs.svelte";
+  import Toastr from "../components/miscComponents/Toastr.svelte";
+  import BuildDetailsCard from "../components/detailCard/BuildDetailsCard.svelte";
```
Comment (bfabiny) [inline ui/src/containers/HtmlHints.svelte]: The folder names should be consistent.
Change
From
    detailCard
To
    detailCardComponents

File: ui/src/containers/ArtilleryReport.svelte [hunk 5]
```diff
@@ -1,26 +1,21 @@
...
-    userApi,
     userSession$,
-    getIgnoreList,
   } from "../stores";
-  import { printTimeDiff, CONSTS } from "../utils/utils";
-  import CardSummary from "../components/CardSummary.svelte";
-  import UpdateArtilleryThreshold from "../components/UpdateArtilleryThreshold.svelte";
-  import ArtilleryDetailsCard from "../components/ArtilleryDetailsCard.svelte";
+  import { CONSTS } from "../utils/utils";
+  import CardSummary from "../components/summaryItem/CardSummary.svelte";
```
Comment (bfabiny) [inline ui/src/containers/ArtilleryReport.svelte]: Folder name should be consistent
Change
From
summaryItem
To
summaryItemComponents

File: ui/src/components/miscComponents/SocialLogin.svelte (hunk 1 above)
Comment (tombui99) [reply ui/src/components/miscComponents/SocialLogin.svelte]: Done - renamed to LoadingCircle

File: ui/src/containers/ArtilleryReport.svelte (hunk 2 above)
Comment (tombui99) [reply ui/src/containers/ArtilleryReport.svelte]: Done - renamed to artillerycomponents

File: ui/src/containers/Dashboard.svelte (hunk 3 above)
Comment (tombui99) [reply ui/src/containers/Dashboard.svelte]: Done - renamed to buildlistcardcomponents

File: ui/src/containers/HtmlHints.svelte (hunk 4 above)
Comment (tombui99) [reply ui/src/containers/HtmlHints.svelte]: Done - renamed to detailcardcomponents

File: ui/src/containers/ArtilleryReport.svelte (hunk 5 above)
Comment (tombui99) [reply ui/src/containers/ArtilleryReport.svelte]: Done - renamed to summaryitemcomponents

**Review by bfabiny — APPROVED** [review]
Excellent, thanks Tom!

**Pull request merged**